spec/syntax.md

```diff
@@ -648,7 +648,10 @@ on the contents of these _annotations_.
 Implementations MUST NOT assign meaning or semantics to
 an _annotation_ starting with `reserved-annotation-start`:
 these are reserved for future standardization.
-Implementations MUST trim whitespace around a _reserved body_,
+Whitespace before or after a _reserved body_ is not part of the _reserved body_.
+Implementations MUST NOT remove or alter the contents of a _reserved body_,
+including any interior whitespace,
+but MAY remove or alter whitespace before or after the _reserved body_.
 but MUST NOT remove or alter the contents of a _reserved body_.
```

Comment:
My suggestion missed this line.

```suggestion
```

Reply:
Already fixed.

Reply:
Thanks for your subtle wording. (I guess I'm not familiar with all the nuances.)